packages/components/src/interfaces/internal/ChildrenPages.ts

```diff
@@ -1,9 +1,9 @@
 import type { Static } from "@sinclair/typebox"
 
-import type { ChildpageMetaSchema } from "~/schemas/childpage"
+import type { ChildpageSchema } from "../complex"
 import type { IsomerSiteProps, LinkComponentType } from "~/types"
 
-export interface ChildrenPagesProps extends Static<typeof ChildpageMetaSchema> {
+export interface ChildrenPagesProps extends Static<typeof ChildpageSchema> {
```

Comment:
question: sorry a bit confused, what's the difference between this `ChildrenPagesProps` and the `ChildpageProps` type? Seems like they can be merged.

Reply:
merged - i also standardized to `childrenpages` ba